### DISCUSSION TITLE
Next release date?

### DISCUSSION THREAD

**Discussion by asyncore** [protectwise/troika#332] [General]
Hello!
I was wondering when the next release will be posted? There's a fix in the main branch we look forward to consuming. Any ETA will be appreciated 🙏

**Comment by lojjic** [comment]
Thanks for the reminder, I'll try to publish a release by the end of the week.

**Comment by lojjic** [comment]
Released 0.50.0 today.

**Comment by asyncore** [comment]
Thank you!
